pymtl3/passes/GenDAGPass.py

```diff
@@ -46,19 +47,15 @@ def _generate_net_blocks( self, top ):
     top._dag.genblk_writes  = {}
     # top._dag.genblk_src     = {}
 
-    # To reduce the time to compile update blocks, I first group the list
-    # of update blocks that have the same host object together and fire
-    # them in a single compile command.
+    # Fall back to compiling one block at a time
 
-    # Note that it introduced problems with multiple update blocks having
-    # the same const writer at the same hostobj. As a result, I add a
-    # suffix to each block like Bits1_xxx_no_12, Bits1_xxx_no_13 to
-    # disambiguate the blocks
-
-    hostobj_allsrc = defaultdict(str)
-    hostobj_bits   = defaultdict(set)
-    blkname_meta   = {}
-    blkname_suffix = {}
+    # TODO see if directly compiling AST instead of source can be faster
```

Comment:
Maybe have a comment here about why we need block-by-block compilation (to differentiate different structs with the same __name__)?